Header header
int32 frame_id
sensor_msgs/CompressedImage image
float32 scale
